core/src/main/java/org/apache/accumulo/core/clientImpl/InstanceOperationsImpl.java

```diff
@@ -215,38 +220,46 @@ public Map<String,String> getSystemProperties()
   }
 
   @Override
+  @Deprecated(since = "4.0.0")
   public List<String> getManagerLocations() {
-    return context.getManagerLocations();
+
+    Set<ServerId> managers = getServers(ServerTypeName.MANAGER);
```

Comment:
This API could be introduced in 3.1, so you can drop the deprecated stuff in 4.0, if you wanted.

Reply:
I don't know that the amount of work that would entail would be worth the trouble.

Reply:
This change depends on change that only exist in main.  So moving it to 3.1 would entail moving a set of changes.  Seems like it would be best to get this into main, then if anyone is interested they can look at moving the whole set of related changes back.